### ISSUE TITLE
[Bug]: [EmbeddingDE] docker run produces ImportError: 'send_event' & 'haystack.telemetry'

### ISSUE THREAD

**Issue by exowanderer** [rti/gbnc#6]
In Branch: [feature/EmbeddingDE](https://github.com/rti/gbnc/tree/feature/EmbeddingDE)

After introducing the new FAISSDoctumentStore (for sqlite) and jina.ai embedding for DE+EN embedding, the `docker run` began to flag the error:

```ImportError: cannot import name 'send_event' from 'haystack.telemetry'```

This error started early in the process, but could not discover how or why.

Following the guidance from https://github.com/deepset-ai/haystack/issues/6652, I began to turn off `farm-haystack` features and/or `haystack-ai` in the Dockerfile. 

Later, I attempted to forcibly install Python3.8 or Python.3.9 -- both suggest by the Haystack:Issue:6652 above. 

These solutions may still exist, but my implementation did not fix the error.

**Comment by exowanderer** [comment]
I have been communicating with the devs about the `send_message` in `haystack.telemetry` bug [Haystack's GitHub Issues](https://github.com/deepset-ai/haystack/issues/6652). 

After I gave them the relevant section of [our Dockerfile](https://github.com/rti/gbnc/blob/main/Dockerfile), they realised that we are simultaneously using Haystack 1.x and Haystack 2.0 Beta, causing ImportErrors.  

In particular,  `haystack-ai` is Haystack 2.0-beta and `farm-haystack` is Haystack 1.x. I think that `ollama-haystack` is also Haystack 2.0 Beta.

For testing, I removed `haystack-ai` completely from the Dockerfile. I am now working through the usual bugs related to changing import versions. Haystack 2.0 does not seem to have an EmbeddingDocumentStore,. From what I can tell, 2.0Beta only has `InMemoryDocumentStore`.
